code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -563,6 +590,185 @@
+		/obj/item/gun/magic/staff/healing = 1,
+		/obj/item/gun/magic/staff/chaos = 2,
+		/obj/item/gun/magic/staff/animate = 2,
+		/obj/item/gun/magic/staff/focus = 2,
+		/obj/item/gun/magic/hook = 1,
+		/obj/item/hierophant_club = 3, //Strong so super costly
+		/obj/item/lava_staff = 2 ) //Hot seller so 2
+	var/obj/item/pickeds = pick(list_s)
+	value += list_s[pickeds]
+	new pickeds(src)
+
+	//Random magical artifact.
+	var/list/list_a = list(
+		/obj/item/necromantic_stone = 2,
+		/obj/item/scrying = 1, //thematic discount
+		/obj/item/organ/internal/heart/cursed/wizard = 1,
+		/obj/item/organ/internal/vocal_cords/colossus/wizard = 2,
+		/obj/item/warp_cube/red = 1,
+		/obj/item/reagent_containers/food/drinks/everfull = 2,
+		/obj/item/clothing/suit/space/hardsuit/shielded/wizard = 2,
+		/obj/item/jacobs_ladder = 1, //funny
+		/obj/item/immortality_talisman = 1 ) //spells recharge when invincible
+	var/obj/item/pickeda = pick(list_a)
+	value += list_a[pickeda]
+	new pickeda(src)
+
+	//Summon
+	switch(rand(1, 8))
+		if(1)
+			new /obj/item/antag_spawner/slaughter_demon(src)
+			value += 2
+		if(2)
+			new /obj/item/antag_spawner/morph(src)
+			value += 1
+		if(3)
+			new /obj/item/antag_spawner/slaughter_demon/laughter(src)
+			value += 1
+		if(4)
+			new /obj/item/antag_spawner/slaughter_demon/shadow(src)
+			value += 1
+		if(5)
+			new /obj/item/antag_spawner/revenant(src)
+			value += 1
+		if(6)
+			new /obj/item/contract(src)
+			value += 2
+		if(7)
+			new /obj/item/guardiancreator(src)
+			value += 1
+		if(8)
+			if(prob(25))
+				new /obj/item/reagent_containers/food/snacks/grown/nymph_pod(src)
+				new /obj/item/slimepotion/sentience(src)
+			else
+				new /obj/item/paicard(src) //Still useful, not a point useful.
+
+	//Treat / potion. Free.
+	var/obj/item/pickedt = pick(
+			/obj/item/storage/box/syndidonkpockets, // Healing + speed
+			/obj/item/reagent_containers/food/drinks/bottle/dragonsbreath, // Killing
+			/obj/item/reagent_containers/food/drinks/bottle/immorality, // Super healing for 20 seconds
+			/obj/item/reagent_containers/food/snacks/meatsteak/stimulating, //Healing + stun immunity
+			/obj/item/reagent_containers/food/snacks/plum_pie ) // Great healing over long period of time
+	new pickedt(src)
+
+
+	if(value > NANNY_MAX_VALUE || value < NANNY_MIN_VALUE)
+		if(attempts >= 5)
+			message_admins("Failed to generate the wizard a properly priced magic nanny bag!")
+		else
+			new /obj/item/storage/backpack/duffel/magic_nanny_bag(get_turf(loc), attempts + 1)
+		qdel(src)
+
+#undef NANNY_MAX_VALUE
+#undef NANNY_MIN_VALUE
+
+/obj/item/reagent_containers/food/drinks/bottle/dragonsbreath
+	name = "flask of dragons breath"
+	desc = "Not recommended for wizardly consumption. Recommended for mundane consumption!"
+	icon_state = "holyflask"
+	color = "#DC0000"
+	volume = 100
+	list_reagents = list("dragonsbreath" = 80, "hell_water" = 20)
+
+/obj/item/reagent_containers/food/drinks/bottle/immorality
+	name = "drop of immorality"
```

Comment:
intended spelling discrepency between here+path, and desc?

Reply:
close, I just can't spell

Reply:
Oh no, this is intended.
A drop of immortal doesn't make sense, nor does the drop "making you immorality"

The drop of immorality will make you immortal temporary. It does not give you immorality, it only makes you "immortal" or a period

Reply:
I am dense